Home.py

```diff
@@ -33,104 +34,238 @@ def load_config(file_path=".streamlit/config.toml"):
+                            color=st.session_state.font
+                        ),  # X-axis title color
+                        tickfont=dict(color=st.session_state.font),  # X-axis tick color
+                    ),
+                    yaxis=dict(
+                        title="Price [p/kWh]",  # Label the y-axis
+                        title_font=dict(
+                            color=st.session_state.font
+                        ),  # Y-axis title color
+                        tickfont=dict(color=st.session_state.font),  # Y-axis tick color
+                    ),
+                )
+                st.plotly_chart(fig_all)
+        elif day_to_plot == "Tomorrow":
+            tomorrow_date = (datetime.now(pytz.UTC) + pd.Timedelta(days=1)).date()
+            df_tom = st.session_state.df[
+                st.session_state.df["valid_from"].dt.date == tomorrow_date
+            ]
+
+            if df_tom.empty:
+                st.write("No data available for tomorrow.")
+            else:
+                fig_all = go.Figure()
+                fig_all.add_trace(
+                    go.Bar(
+                        x=df_tom["valid_from"],  # Time column
+                        y=df_tom["value_inc_vat"],  # Price column
+                        marker=dict(
+                            color=st.session_state.marker
+                        ),  # Optional: Set bar color
+                        name="Price [p/kWh]",  # Legend name
+                    )
+                )
+
+                fig_all.update_layout(
+                    plot_bgcolor=st.session_state.bg_color,  # Set the plot area background to white
+                    font=dict(
+                        color=st.session_state.font,  # Set text color
+                        size=14,  # Optional: Adjust text size for better visibility
+                    ),
+                    title=dict(
+                        text="Agile Pricing",  # Add a title
+                        font=dict(color=st.session_state.font),  # Set title text color
+                    ),
+                    xaxis=dict(
+                        title="Time",  # Label the x-axis
+                        title_font=dict(
+                            color=st.session_state.font
+                        ),  # X-axis title color
+                        tickfont=dict(color=st.session_state.font),  # X-axis tick color
+                    ),
+                    yaxis=dict(
+                        title="Price [p/kWh]",  # Label the y-axis
+                        title_font=dict(
+                            color=st.session_state.font
+                        ),  # Y-axis title color
+                        tickfont=dict(color=st.session_state.font),  # Y-axis tick color
+                    ),
+                )
+                st.plotly_chart(fig_all)
+
+
+def get_optimal_coffee_time(df, current_time):
+    # Get the date for tomorrow if it is after 10am, otherwise use today
```

Comment:
Can we add to the docstring here to include the default mass and temperature used for the calculation?

```suggestion
    """
    Uses the date as tomorrow if it is after 10am, otherwise use today. Default mass, temperature is 650ml, 17degC
    """
```